src/v2/api/controllers/ScheduleController.ts

```diff
@@ -375,4 +381,44 @@ export class ScheduleController {
       }
     }
   }
+
+  @UseGuards(TelegramGuard)
+  @ApiBearerAuth()
+  @ApiQuery({
+    name: 'week',
+    required: false,
+  })
+  @ApiOkResponse({
+    type: FortnightGeneralEventsResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidGroupIdException:
+      Group with such id is not found
+      
+    DataNotFoundException:
+      Data was not found`,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @Get('/groups/:groupId/general/fortnight')
+  async getGeneralFortnightEvents (
+    @Param('groupId', GroupByIdPipe) groupId: string,
+    @Query('week') week: number,
+  ) {
+    const result = await this.scheduleService.getGeneralFortnightEvents(groupId, week);
+
+    return {
+      eventsOfFirstWeek: this.scheduleMapper.getEvents(result.eventsOfFirstWeek),
+      eventsOfSecondWeek: this.scheduleMapper.getEvents(result.eventsOfSecondWeek),
```

Comment:
In this case I`d rather shorten: 
* `eventsOfFirstWeek` to `firstWeekEvents`;
* `eventsOfSecondWeek` to `secondWeekEvents`.